test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,151 @@
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid prefix or netmask')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "1.2.3.4.5")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid address 1.2.3.4.5')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "10.0.0.1/24")
+
+        # peer
+        b.click("button:contains('Add peer')")
+        b.wait_visible("#network-wireguard-settings-peer-0")
+        b.set_input_text("#network-wireguard-settings-publickey-peer-0", m2_pubkey)
+        b.set_input_text("#network-wireguard-settings-endpoint-peer-0", f"192.168.100.12:{m2_port}")
+        b.set_input_text("#network-wireguard-settings-allowedips-peer-0", "10.0.0.2")
+        b.click("button:contains('Add peer')")
+        b.wait_visible("#network-wireguard-settings-peer-1")
+        b.set_input_text("#network-wireguard-settings-publickey-peer-1", m2_pubkey)
+        b.set_input_text("#network-wireguard-settings-endpoint-peer-1", f"192.168.100.12:{m2_port}")
+        b.set_input_text("#network-wireguard-settings-allowedips-peer-1", "10.0.0.2")
+        b.click("button#network-wireguard-settings-btn-close-peer-0")
+        b.wait_not_present("#network-wireguard-settings-peer-1")
+
+        b.click("button:contains('Save')")
+        b.wait_not_present("#network-wireguard-settings-dialog")
+
+        m2.execute(f"wg set wg0 peer {m1_pubkey} allowed-ips 10.0.0.1/32") # endpoint and port is not necessary for a peer if that peer estalishes the connectio first (i.e. the client)
+
+        # check connection over ipv4
+        try:
+            m1.execute("ping -c 5 10.0.0.2")
+        except:
+            m1.execute("ip a >&2; ip route >&2; nmcli c >&2; wg >&2")
+            m2.execute("ip a >&2; ip route >&2; nmcli c >&2; wg >&2")
+            raise
+
+        # check connection over ipv6
+        m2.execute("ip addr add 2001::2/64 dev wg0")
+        m2.execute(f"wg set wg0 peer {m1_pubkey} allowed-ips 10.0.0.1/32,2001::1")
+
+        b.click("#networking-interfaces button:contains('wg0')")
+        b.click("#networking-edit-ipv6")
+        b.wait_visible("#network-ip-settings-dialog")
+        b.select_from_dropdown("#network-ip-settings-select-method", "manual")
+        b.set_input_text("#network-ip-settings-address-0", "2001::1")
+        b.set_input_text("#network-ip-settings-netmask-0", "64")
+        b.set_input_text("#network-ip-settings-gateway-0", "::")
+        b.click("button:contains('Save')")
+        b.wait_not_present("#network-ip-settings-dialog")
+        b.click("#networking-edit-wg")
+        b.wait_visible("#network-wireguard-settings-dialog")
+        b.set_input_text("#network-wireguard-settings-allowedips-peer-0", "10.0.0.1/32,2001::2")
+        b.click("button:contains('Save')")
+        b.wait_not_present("#network-wireguard-settings-dialog")
+        with b.wait_timeout(60):
```

Comment:
Did you really see this time out with 15s? That seems absurdly long for setting an address.